Description: Autogenerated patch header for a single-debian-patch file.
 The delta against upstream is either kept as a single patch, or maintained
 in some VCS, and exported as a single patch instead of more manageable
 atomic patches.
Forwarded: not-needed

---
--- notmuch-0.38.3.orig/test/T350-crypto.sh
+++ notmuch-0.38.3/test/T350-crypto.sh
@@ -453,6 +453,7 @@ y
     | gpg --no-tty --quiet --import
 output=$(notmuch show --format=json --verify subject:"test signed message 001" \
     | notmuch_json_show_sanitize \
+    | sed -e 's/"key-\(revoked\|missing\)"/"key-revoked"/g' \
     | sed -e 's|"created": [1234567890]*|"created": 946728000|')
 expected='[[[{"id": "XXXXX",
  "match": true,
--- notmuch-0.38.3.orig/test/test-lib.el
+++ notmuch-0.38.3/test/test-lib.el
@@ -33,6 +33,9 @@
 
 (require 'cl-lib)
 
+;; Use old pretty print algorithm, so tests don't break with Emacs 30
+(setq-default pp-default-function 'pp-28)
+
 ;; Ensure that the dynamic variables that are defined by this library
 ;; are defined by the time that we let-bind them.  This is needed
 ;; because starting with Emacs 27 undeclared variables in evaluated
